services/net/contentmigration/Extensions/NewsItemExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using TNO.Services.ContentMigration.Sources.Oracle;
+
+namespace TNO.Services.ContentMigration.Extensions;
+
+/// <summary>
+/// DictionaryExtensions static class, provides extension methods for dictionary objects.
+/// </summary>
+public static class DictionaryExtensions
+{
+    /// <summary>
+    /// Get the value from the dictionary for the specified 'key' and return it as an int.
+    /// </summary>
+    /// <param name="dict"></param>
+    /// <param name="key"></param>
+    /// <param name="defaultValue"></param>
+    /// <returns></returns>
+    public static int GetIntValue(this IDictionary<string, Microsoft.Extensions.Primitives.StringValues> dict, string key, int defaultValue = 0)
```

Comment:
I'm certain we already have this extension in the Core library.

Reply:
bad copy paste...